### PR TITLE
[one-cmds] Add constant.py dependency

### PR DESCRIPTION
**Pull request by jinevening**
This adds constant.py to cmake dependency for automatic recompilation.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

### CODE REVIEW

File: compiler/one-cmds/CMakeLists.txt [hunk 1]
```diff
@@ -168,7 +169,7 @@ add_custom_command(OUTPUT ${O1_CFG_FILE_BIN}
           --constant ${O1_OPTION}
           --format cfg
           --output_path ${O1_CFG_FILE_BIN}
-  DEPENDS ${CONSTANT_EXPORTING_SCRIPT}
+  DEPENDS ${CONSTANT_EXPORTING_SCRIPT} ${CONSTANT_SCRIPT}
```
Comment (jinevening) [inline compiler/one-cmds/CMakeLists.txt:172]: `export_constant.py` internally uses `constant.py`, but it was not considered. So, even though O1 list in `constant.py` is updated, the actual O1 list in the `O1.cfg` is not updated if there is a previously-built file.

This PR fixes the issue, so now O1 list is updated when `constant.py` is updated.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by zetwhite — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**